File: Counting Divisors/Counting_Divisors.cpp
#include<bits/stdc++.h>
using namespace std;
 
#define ll long long
#define int ll
const int mod = 1000000007;
const int lim = 1e6 + 1;
int divs[lim];
 
signed main()
{
	int test, n, i, j;
	cin >> test;
	for(i = 1; i < lim; i++)
	{
	    for(j = i; j < lim; j += i)
	    {
	        divs[j]++;
	    }
	}
	
	while(test--)
	{
		cin >> n;
		cout << divs[n] << endl;
	}

File: Counting Divisors/Counting_Divisors.py
mod = 10**9 + 7
lim = 10**6 + 1
 
divs = [0] * lim
for i in range(1, lim):
    for j in range(i, lim, i):
        divs[j] += 1
 
test = int(input())
for _ in range(test):
    n = int(input())
    print(divs[n])
